core/modules/commands/savewikifolder.js

```diff
@@ -35,13 +35,27 @@ Command.prototype.execute = function() {
 	if(this.params.length < 1) {
 		return "Missing wiki folder path";
 	}
-	var wikifoldermaker = new WikiFolderMaker(this.params[0],this.params[1],this.commander);
+	var reExplodePlugins = /^\[(.)*\]$/; // Test Filter Features [ ] Regular Expressions
+	switch (this.params.length){
+		case 2:
+			if (reExplodePlugins.test(this.params[1])) {
+				this.wikiFilter = this.params[1];
+			} else {
+				this.explodeplugins = this.params[1] === "explodeplugins=no" ? true : false;
+			}
+			break;
+		case 3:
+			this.wikiFilter = this.params[1];
+			this.explodeplugins = this.params[2] === "explodeplugins=no" ? true : false;
+	}
+	var wikifoldermaker = new WikiFolderMaker(this.params[0],this.wikiFilter,this.explodeplugins,this.commander);
 	return wikifoldermaker.save();
 };
 
-function WikiFolderMaker(wikiFolderPath,wikiFilter,commander) {
+function WikiFolderMaker(wikiFolderPath,wikiFilter,explodeplugins,commander) {
```

Comment:
IMO that's not backwards compatible. The new function parameter needs to be at the end. As is it can invalidate code, that imports the Command.

Reply:
Okay, I'll revise it.